name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/HTMLTableQuoteFeed.java

```diff
@@ -219,6 +129,32 @@ void setValue(Element value, LatestSecurityPrice price, String languageHint) thr
         }
     }
 
+    private static class VolumeColumn extends Column
+    {
+        @SuppressWarnings("nls")
+        public VolumeColumn()
+        {
+            super(new String[] { "Volume.*", "Umsatz" , "Stücke" });
+        }
+
+        @Override
+        public void setValue(Element value, LatestSecurityPrice price, String languageHint) throws ParseException
+        {
+            try
+            {
+                if ("-".equals(value.text().trim())) //$NON-NLS-1$
+                    price.setVolume((int) LatestSecurityPrice.NOT_AVAILABLE);
+                else
+                    price.setVolume(super.asInt(value));
+            }
+            catch (Throwable ex)
+            {
+                System.err.println("Uncaught exception - " + ex.getMessage());
+                ex.printStackTrace(System.err);
```

Comment:
Which exception ends up here? The "asInt" already catches Throwables. 
And: if we catch the exception, it makes sense the set the value to NOT_AVAILABLE.